cprover_bindings/src/irep/irep_id.rs

```diff
@@ -1462,6 +1463,7 @@ impl ToString for IrepId {
             IrepId::CSpecLoopInvariant => "#spec_loop_invariant",
             IrepId::CSpecRequires => "#spec_requires",
             IrepId::CSpecEnsures => "#spec_ensures",
+            IrepId::CSpecAssigns => "#spec_assigns",
```

Comment:
I'd remove everything related to assigns clauses from this pull request, which is focus only on "simple requires and ensures".